### PR TITLE
Add full screen off canvas menu building block

### PR DESCRIPTION
**Pull request by brettsmason**
This adds a full width overlay menu using off canvas as a base.

### CODE REVIEW

File: src/building-blocks/off-canvas-full-screen/off-canvas-full-screen.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: 'Full Screen Off Canvas'
+tags:
+  - off-canvas
```
Comment (rafibomb) [inline src/building-blocks/off-canvas-full-screen/off-canvas-full-screen.yml]: add tags
- nav
- menu

**Comment by rafibomb** [comment]
@brettsmason This is great! I want to add it in for Monday. Would you like me to do the small edits?

**Comment by brettsmason** [comment]
@rafibomb Sorry for the delay. Would you mind? I'm away at the moment without proper computer access and wont be back until Tuesday evening.

**Comment by rafibomb** [comment]
Merged in! https://github.com/zurb/building-blocks/commit/8e178c7d94b51890ad8f81af2be49167a593877e

**Pull request closed without merging**